openpype/scripts/ocio_wrapper.py

```diff
@@ -96,11 +96,41 @@ def _get_colorspace_data(config_path):
 
     config = ocio.Config().CreateFromFile(str(config_path))
 
-    return {
-        c.getName(): c.getFamily()
-        for c in config.getColorSpaces()
+    colorspace_data = {
+        color.getName(): {
+            "type": "colorspace",
+            "family": color.getFamily(),
+            "categories": list(color.getCategories()),
+            "aliases": list(color.getAliases()),
+            "equalitygroup": color.getEqualityGroup(),
+        }
+        for color in config.getColorSpaces()
     }
 
+    # add looks
+    looks = config.getLooks()
+    if looks:
+        colorspace_data.update({
+            look.getName(): {
+                "type": "look",
+                "process_space": look.getProcessSpace()
+            }
+            for look in looks
+        })
+
+    # add roles
+    roles = config.getRoles()
+    if roles:
+        colorspace_data.update({
+            role[0]: {
+                "type": "role",
+                "colorspace": role[1]
+            }
+            for role in roles
```

Comment:
Does `roles` provide more than just index zero and one or is it a two-tuple? If a two-tuple it might be more readable as:
```suggestion
            role: {
                "type": "role",
                "colorspace": colorspace
            }
            for (role, colorspace) in roles
```